src/main/java/edu/SlowestKeySolution.java

```
@@ -0,0 +1,35 @@
+package edu;
+
+public class SlowestKeySolution {
+
+    public static final char INVALID = '-';
+
+    public static char solve(int[] releaseTimes, String keysPressed) {
+        if (releaseTimes != null && releaseTimes.length != 0 &&
```

invert if statement and return invalid in case input is invalid.
Fail early.

---

This will reduce nesting